src/cms/views/pages/page_tree_view.py

```diff
@@ -4,23 +4,17 @@
 from django.shortcuts import render, redirect
 from django.utils.decorators import method_decorator
 from django.utils.translation import ugettext as _
-
 from django.views.generic import TemplateView
 
+from ...constants import translation_status
 from ...decorators import region_permission_required
+from ...forms.pages.page_filter_form import PageFilterForm
 from ...models import Region, Language
-from .page_mixin import PageMixin
 
 
 @method_decorator(login_required, name="dispatch")
 @method_decorator(region_permission_required, name="dispatch")
-# pylint: disable=too-many-ancestors
-class PageTreeView(PermissionRequiredMixin, TemplateView, PageMixin):
-    """
-    View for showing the page tree
-    """
-
-    #: Required permission of this view (see :class:`~django.contrib.auth.mixins.PermissionRequiredMixin`)
+class PageTreeView(PermissionRequiredMixin, TemplateView):
```

Comment:
```suggestion
# pylint: disable=too-many-ancestors
class PageTreeView(PermissionRequiredMixin, TemplateView, PageMixin):
    """
    View for showing the page tree
    """

    #: Required permission of this view (see :class:`~django.contrib.auth.mixins.PermissionRequiredMixin`)
```
Please do not remove code without asking :wink: 
The `PageMixin` is required for the confirmation popups when archiving or deleting pages from the page tree.

Reply:
Mhm, this wasn't on purpose. I can't remember changing this. Now, I changed it back